src/getDataSet.js

```diff
@@ -2,9 +2,9 @@ const getVR = require('./getVR')
 const getValue = require('./getValue')
 const Tags = require('./Tags')
 
-const attributeToJS = async (metadata, tag, dataSet, attr, callback, options) => {
+const attributeToJS = async (metadata, tag, dataSet, attr, callback, options, parentAttr) => {
```

Comment:
Can you just add it into the options?  There may well be places that there are additional values to add into options such as the time zone, specific character set etc, and there shouldn't be an indefinite number of arguments to add here.

Reply:
Options is actually the actual state of program (in this case its commander options). TBH we dont really need that once program has already parsed the options (we dont really need callback param either). This is actually a clear example of  code smell (long parameter list and inappropriate intimacy), but to preserve layers above and bellow I have added this parentAttr (what I am envisioning here is to clean up this smell on future tasks).
So what do you prefer?
1. Do your request
2. Do the clean up now (i.e decouple callback and options params)
3. Do the clean up later (i.e decouple callback and options params) and keep parentAttr

Reply:
Lets not do #1, as I agree, its a code smell to have that mixed up.  I'd rather do the cleanup as a separate task, so just do #3, but plan on doing a cleanup phase after this commit.